src/main/java/frc/robot/Constants.java

```diff
@@ -51,6 +51,9 @@ public class Arm {
         // PID Profile constants
         public static final double MAXIMUM_VELOCITY = 150; // degrees per second
         public static final double MAXIMUM_ACCELERATION = 750; // degrees per second squared
+
+    		public static final double LONG_SHOT_ANGLE = 94;
```

Comment:
There are tabs in this line, set up your IDE to not use tabs!

Reply:
accidental manual tab because my indentation is set to 2 and i just wanted it to get over there

Reply:
i'll remove it

Reply:
your editor should have some way to make pressing tab just put spaces idk